### ISSUE TITLE
can not complete class name after extends or implements. get error when complete after `(`

### ISSUE THREAD

**Issue by wsdjeg** [artur-shaik/vim-javacomplete2#179]
example:

```java
 class Foo extends L_  //_ is cursor
```

```java
 class Foo implements L_
```
```java
class Foo{
   void run(){
   this.wait(_   //here type ctrl+x ctrl+o will get error


}
}
```

```
Error detected while processing function javacomplete#Complete..javacomplete#complete#complete#Complete..javacomplete#complete#context#FindContex
t:
line  139:
E117: Unknown function: s:IsKeyword
Press ENTER or type command to continue
Error detected while processing function javacomplete#Complete..javacomplete#complete#complete#Complete..javacomplete#complete#context#FindContex
t:
line  139:
E15: Invalid expression: statement[pos-1] == '.' && !s:IsKeyword(strpart(statement, pos))
```

**Comment by artur-shaik** [comment]
Fixed in fc93d16695349393cbd89d43b0465d186b21c771

**Comment by wsdjeg** [comment]
Thanks everything ,the plugin works well.
